### PR TITLE
usearch: Make installed binary executable

### PR DESCRIPTION
**Pull request by erimar77**

### CODE REVIEW

**Comment by erimar77** [comment]
It looks like the install method just preserves the src permissions.  This is a manual download, so we can't be sure what the permissions will be. In my case, the file was saved with 644, resulting in a non-functioning package.

File: var/spack/repos/builtin/packages/usearch/package.py [hunk 1]
```diff
@@ -28,3 +28,5 @@ def install(self, spec, prefix):
         mkdirp(prefix.bin)
         install('usearch{0}_i86linux32'.format(self.version),
                 prefix.bin.usearch)
+        chmod = which('chmod')
+        chmod('+x', join_path(prefix.bin, 'usearch'))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/usearch/package.py]: You can actually use `prefix.bin.usearch` here, no `join_path` necessary.

**Comment by trecakov** [comment]
@erimar77 
Hi, I am getting this error. The downloaded file is there with the correct name.

=> Installing usearch
==> Searching for binary cache of usearch
==> Warning: No Spack mirrors are currently configured
==> No binary for usearch found: installing from source
==> Fetching file:///home/pkgmgr/usearch11.0.667_i86linux32
curl: (37) Couldn't open file /home/pkgmgr/usearch11.0.667_i86linux32
==> Fetching from file:///home/pkgmgr/usearch11.0.667_i86linux32 failed.
==> Error: FetchError: All fetchers failed for usearch-11.0.667-f7t2kwkhsnhm75nxw62razuzd77mdzkt

/software/packages/spack/lib/spack/spack/package.py:916, in do_fetch:
        913                raise FetchError("Will not fetch %s" %
        914                                 self.spec.format('$_$@'), ck_msg)
        915
  >>    916        self.stage.create()
        917        self.stage.fetch(mirror_only)
        918        self._fetch_time = time.time() - start_time
        919

Any suggestions?

**Comment by erimar77** [comment]
@trecakov The only version listed is 10.0.240, can you try using that executable?  You'll also need to modify the md5sum to match your file, as each download has your license information baked into it.

**Comment by trecakov** [comment]
@erimar77 I added version 11.0.667 in the package.py.
Anyways, I tried with 10.0.240 and it worked. Thanks

**Pull request merged**